ParquetClassLibrary/Characters/Being.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using System.Collections.Generic;
+using ParquetClassLibrary.Sandbox.IDs;
+using ParquetClassLibrary.Utilities;
+
+namespace ParquetClassLibrary.Characters
+{
+    /// <summary>
+    /// Models the basic definitions shared by any in-game actor.
+    /// </summary>
+    public abstract class Being : Entity
+    {
+        #region Characteristics
+        /// <summary>The <see cref="Biome"/> in which this character is at home.</summary>
+        public Biome NativeBiome { get; set; }
+
+        /// <summary>The <see cref="Behavior"/> governing the way this character acts.</summary>
+        public Behavior PrimaryBehavior { get; set; }
+
+        /// <summary>Types of parquets this critter avoids, if any.</summary>
+        public readonly List<EntityID> Avoids = new List<EntityID>();
+
+        /// <summary>Types of parquets this critter seeks out, if any.</summary>
+        public readonly List<EntityID> Seeks = new List<EntityID>();
```

Comment:
comment: Fascinating! The flexibility shown here is really interesting. Food, companions, the player themselves. This set of fields can model simple behavior quickly, in a way that is pretty captivating.

Reply:
Yay, thanks!